osquery/tables/system/posix/smbios_utils.cpp

```diff
@@ -142,5 +148,21 @@ std::string dmiString(uint8_t* data, uint8_t* address, size_t offset) {
 
   return std::string(bp);
 }
+
+std::string dmiBitFieldToStr(size_t bitField,
+                             const std::map<uint8_t, std::string>& table) {
+  std::stringstream ss;
+
+  for (uint8_t i = 0; i < table.size(); i++) {
+    if (1 << i & bitField) {
+      ss << table.at(i) + " ";
+    }
+  }
+
+  auto result = ss.str();
+  boost::algorithm::trim(result);
```

Comment:
If `!ss.empty()` then you can pop the last character instead of trimming?

Reply:
Yeah I can.  Good catch.